src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -488,41 +506,32 @@ export class PoolFormComponent implements OnInit {
     this.triggerApiTask(pool);
   }
 
-  private extendByItemsForSubmit(pool, items: any[]) {
-    items.forEach((item) => this.extendByItemForSubmit(pool, item));
+  /**
+   * Retrieves the values for the given form field descriptions and assigns the values to the given
+   * object. This method differentiates between `add` and `edit` mode and acts differently on one or
+   * the other.
+   */
```

Comment:
This comment makes more sense in line 518 above `assignFormField` as `assignFormFields` will always act the same way by calling `assignFormField`.

Reply:
If I hit Ctrl+q, I will get the complete documentation of a method without having to go and look at it's source. If the comment was elsewhere, I wouldn't be able to do so. And this method is called very often in that controller. I think it's most helpful where it currently is.

Reply:
Than at least add it to the other method as well. As it is not only used in `assignFormFields`.

Reply:
We agreed on having both methods commented.